components/search/index.css

```diff
@@ -193,6 +208,8 @@
 	block-size: var(--mod-search-block-size, var(--spectrum-search-block-size));
 	padding-block-start: calc(var(--mod-search-top-to-text, var(--spectrum-search-top-to-text)) - var(--mod-search-border-width, var(--spectrum-search-border-width)));
 	padding-block-end: calc(var(--mod-search-bottom-to-text, var(--spectrum-search-bottom-to-text)) - var(--mod-search-border-width, var(--spectrum-search-border-width)));
+	padding-inline-start: calc(var(--mod-search-edge-to-visual, var(--spectrum-search-edge-to-visual)) - var(--mod-search-border-width, var(--spectrum-search-border-width)) + var(--mod-search-icon-size, var(--spectrum-search-icon-size)) + var(--mod-search-text-to-icon, var(--spectrum-search-text-to-icon)));
+	padding-inline-end: calc(var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size)) - var(--mod-search-border-width, var(--spectrum-search-border-width)));
```

Comment:
Is this the piece of padding between the text input and the edge?

I noticed we're not using `component-pill-edge-to-text` tokens here and I'm wondering why? Is it because we're trying to leave more space for the clear button?

But also the clear button isn't really spec'd inside of search at all, which feels odd to me, is there a reason for that, that you know of?

Reply:
Yes- the `padding-inline-start` should add space between the value & placeholder text and the search icon, and then the `padding-inline-end` should give space between the value in the input and the clear button (that's when it's most noticeable I think).

The `--spectrum-search-edge-to-visual` should be set to the component pill edge to text tokens for each t-shirt size. But if you're asking why aren't we using that in the `padding-inline-end`, I don't have a good answer for that. I'll have to get back with you.

I don't know why the clear button isn't spec'd, but I can certainly reach out to design and ask. All I can infer from the designs is that the clear button is all the way at the end of the search container. That's how the component seems to be styled currently as well.